sonar-project.properties

```diff
@@ -1,3 +1,3 @@
 sonar.projectKey=fh-statsc
```

Comment:
This file can be removed - we no longer use sonarqube for code analysis

Reply:
@damienomurchu done.